src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBaseKernels.hpp

```diff
@@ -312,10 +310,9 @@ struct AccumulationKernel
           arrayView1d< globalIndex const > const & dofNumber,
           arrayView1d< integer const > const & elemGhostRank,
           arrayView1d< real64 const > const & volume,
-          arrayView1d< real64 const > const & porosityOld,
-          arrayView1d< real64 const > const & porosityRef,
-          arrayView2d< real64 const > const & pvMult,
-          arrayView2d< real64 const > const & dPvMult_dPres,
+          arrayView2d< real64 const > const & porosityOld,
+          arrayView2d< real64 const > const & porosityNew,
+          arrayView2d< real64 const > const & dPoro_dPres,
```

Comment:
It is great that you were able to get rid of `pvMult` with your new developments!